pkg/transparentproxy/transparentproxy_v2.go

```diff
@@ -198,42 +188,72 @@ func (tp *TransparentProxyV2) Setup(tpConfig *config.TransparentProxyConfig) (st
 	return Setup(cfg)
 }
 
-func parseExcludePortsForUIDs(excludeOutboundPortsForUIDs []string, protocol string) ([]config.UIDsToPorts, error) {
+func parseExcludePortsForUIDs(excludeOutboundPortsForUIDs []string) ([]config.UIDsToPorts, error) {
 	var uidsToPorts []config.UIDsToPorts
 	for _, excludePort := range excludeOutboundPortsForUIDs {
 		parts := strings.Split(excludePort, ":")
-		if len(parts) != 2 {
-			return nil, errors.New("value contains too many \":\" - format for excluding ports by UIDs ports:uids")
+		if len(parts) == 0 || len(parts) > 3 {
+			return nil, fmt.Errorf("value: '%s' is invalid - format for excluding ports by UIDs <protocol:>?<ports:>?uids", excludePort)
+		}
+		var portValuesOrRange string
+		var protocolOpts string
+		var uidValuesOrRange string
+		switch len(parts) {
+		case 1:
+			protocolOpts = "*"
+			portValuesOrRange = "*"
+			uidValuesOrRange = parts[0]
+		case 2:
+			protocolOpts = "*"
+			portValuesOrRange = parts[0]
+			uidValuesOrRange = parts[1]
+		case 3:
+			protocolOpts = parts[0]
+			portValuesOrRange = parts[1]
+			uidValuesOrRange = parts[2]
+		}
+		if uidValuesOrRange == "*" {
+			return nil, errors.New("can't use wildcard '*' for uids")
+		}
+		if portValuesOrRange == "*" || portValuesOrRange == "" {
+			portValuesOrRange = "0-65535"
 		}
-		portValuesOrRange := parts[0]
-		uidValuesOrRange := parts[1]
 
-		if err := validatePorts(portValuesOrRange); err != nil {
+		if err := validateUintValueOrRange(portValuesOrRange); err != nil {
 			return nil, err
 		}
 
-		if err := validateUids(uidValuesOrRange); err != nil {
+		if strings.Contains(uidValuesOrRange, ",") {
+			return nil, fmt.Errorf("uid entry invalid:'%s', it should either be a single item or a range", uidValuesOrRange)
```

Comment:
```suggestion
			return nil, fmt.Errorf("UID entry invalid: '%s', it should either be a single item or a range", uidValuesOrRange)
```
Nit: In some places we are using `uid` and in other `UID`. I think we should be consistent.